website/client/src/components/groups/groupPlan.vue

```diff
@@ -100,153 +102,53 @@
           </div>
         </div>
       </div>
+      <!-- Create a new group -->
       <div
         v-if="!upgradingGroup._id"
         class="container col-6 offset-3 create-option"
       >
         <div class="row">
           <h1 class="col-12 text-center purple-header">
-            Create your Group today!
+            Create Your Group Today!
```

Comment:
Do you think it'd be worth tokenizing these strings? We haven't prioritized group plans for non-English teams, but we have it sort of partially done... (is ok if you don't want to do it here)

Reply:
Yes, it would be worth it--should that be included in the upcoming revamp of this page?